File: Examples/CSharp/Articles/ChangeHtmlLinkTarget.cs
using System;
using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    class ChangeHtmlLinkTarget
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            Workbook workbook = new Workbook(sourceDir + "sampleChangeHtmlLinkTarget.xlsx");

            HtmlSaveOptions opts = new HtmlSaveOptions();
            opts.LinkTargetType = HtmlLinkTargetType.Self;

            workbook.Save(outputDir + "outputChangeHtmlLinkTarget.html", opts);

            Console.WriteLine("ChangeHtmlLinkTarget executed successfully.\r\n");
        }
    }
}

File: Examples/VisualBasic/Articles/ChangeHtmlLinkTarget.vb
Imports System.IO
Imports Aspose.Cells

Namespace Articles
    Public Class ChangeHtmlLinkTarget
        Public Shared Sub Run()
            ' ExStart:1
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            Dim inputPath As String = dataDir & "Sample1.xlsx"
            Dim outputPath As String = dataDir & "Output.out.html"

            Dim workbook As Workbook = New Workbook(inputPath)
            Dim opts As HtmlSaveOptions = New HtmlSaveOptions()
            opts.LinkTargetType = HtmlLinkTargetType.Self

            workbook.Save(outputPath, opts)
            Console.WriteLine("File saved: {0}", outputPath)
            ' ExEnd:1

        End Sub
    End Class
End Namespace
